### DISCUSSION TITLE
Exon numbering and transcript segments

### DISCUSSION THREAD

**Discussion by ahwagner** [HGVSnomenclature/hgvs-nomenclature#14] [HGVS nomenclature proposals]
Numbering of exons is an [open HGVS issue](https://varnomen.hgvs.org/recommendations/open-issues/). Exon notation for transcript segments is a critical feature of current laboratory practice for the description of gene fusions, and there is a strong community desire to standardize this.

The VICC gene fusion [transcript segment nomenclature](https://fusions.cancervariants.org/en/latest/nomenclature.html#transcript-sequence-components) provides a new letter prefix for exon-based numbering systems, follows the guidance provided in the open HGVS issue for exon numbering, and is generally aligned with current character conventions (`_` for ranges, `+/-` for offsets) and fusion transcript recommendations (`::` for junctions between HGVS descriptions for each transcript segment). The VICC recommendations use the cDNA alphabet for representation of linker sequences in the exon coordinate space. We would like to resolve any misalignments and incorporate this system into the HGVS recommendations.

**Current status:**
Addressed in https://github.com/HGVSnomenclature/VarNomen/pull/34

**Comment by jtdendunnen** [comment]
The current opinion of the HVNC has been that variants should be described based on positions in a reference sequence. Exon numbers are indirect indicators of a position, are confusing when a gene has several annotated transcripts and should therefor not be used to describe variants. I suggest the HVNC moves the topic to "Closed discussions"..

**Comment by murphyte** [comment]
I also generally discourage use of exon numbers. They can be used if an explicit transcript context is provided. Formally that should also include the genomic sequence and annotation release. But without context, for example referring to BRCA1 exon 5, cannot be reliably interpreted.

**Reply by jfjlaros** [reply]
I would suggest to tie the annotation to a reference *transcript* instead of a genome build. This ensures that exon numbering is stable across builds as it prevents problems introduced by the mapping artifacts (artificial micro-introns) we touched upon earlier.
